### DISCUSSION TITLE
Is there a mechanism to avoid overlapping dialogs/popovers?

### DISCUSSION THREAD

**Discussion by candidia** [ariakit/ariakit#4134] [Questions] [answered]
Wondering if there's something built in that allows two open dialogs/popovers/tooltips/etc to reposition themselves if overlapping?

**Comment by benrodrs** [comment, marked as answer]
As far as I know, the library doesn't have such a feature built in.

**Reply by candidia** [reply]
Thanks I see in the underlying lib there was some discussion but not sure if it ever materialized and what's available there https://github.com/floating-ui/floating-ui/discussions/2715
